pkg/runctl/run_manager_test.go

```diff
@@ -275,28 +314,28 @@ func Test_RunManager_setupStaticNetworkPolicies_Calls_setupNetworkPolicyFromConf
 	expectedError := errors.New("some error")
 	examinee.testing.setupNetworkPolicyFromConfigStub = func(ctx *runContext) error {
 		methodCalled = true
-		assert.Equal(t, runNamespaceName, ctx.runNamespace)
+		assert.Equal(t, h.namespace1, ctx.runNamespace)
 		return expectedError
 	}
 
 	// EXERCISE
 	resultError := examinee.setupStaticNetworkPolicies(runCtx)
 
 	// VERIFY
-	assert.ErrorContains(t, resultError, "failed to set up the configured network policy in namespace \""+runNamespaceName+"\": ")
+	assert.ErrorContains(t, resultError, "failed to set up the configured network policy in namespace \""+h.namespace1+"\": ")
 	assert.Assert(t, errors.Cause(resultError) == expectedError)
 	assert.Assert(t, methodCalled == true)
 }
 
-func Test_RunManager_setupNetworkPolicyThatIsolatesAllPods(t *testing.T) {
+func Test__runManager_setupNetworkPolicyThatIsolatesAllPods(t *testing.T) {
 	t.Parallel()
 
 	// SETUP
 	const (
-		runNamespaceName   = "runNamespace1"
 		expectedNamePrefix = "steward.sap.com--isolate-all-"
 	)
-	runCtx := &runContext{runNamespace: runNamespaceName}
+	h := newTestHelper1(t)
```

Comment:
Why is this better?

Reply:
It structures test helpers (functions and values) instead of polluting the package namespace with lots of individual things where you can't see what belongs together.